src/styles/index.js

```diff
@@ -24,6 +24,7 @@ const UIModal = require('./modal/index.css')
 const UIPagination = require('./pagination/index.css')
 const UIPopover = require('./popover/index.css')
 const UIRadioGroup = require('./radio-group/index.css')
+const UISecondLinearProgress = require('./linear-progress/secondary-linear-progress/index.css')
```

Comment:
Mudaria o nome dessa variavel pra manter o padrão:

```suggestion
const UISecondaryLinearProgress = require('./linear-progress/secondary-linear-progress/index.css')
```

Reply:
Fixed :heavy_check_mark: